python/social_bot/models/database.config

```diff
@@ -1,13 +1,26 @@
 <?xml version='1.0'?>
 <database>
-  <name>Models used by SocialBot</name>
+  <name>Models used by SocialBot, copied or adapted from https://bitbucket.org/osrf/gazebo_models</name>
```

Comment:
not all of them are from https://bitbucket.org/osrf/gazebo_models.
 ball, texture_ground_plane, cartpole are created by us
 pr2 and youbot_noplugin are from somewhere else

Reply:
pr2 and yobot are also copied from there...  iCub is from https://github.com/robotology/icub-gazebo